e2e/testdata/fn-render/validate-generated-resource/.expected/diff.patch

```diff
@@ -0,0 +1,104 @@
+diff --git a/db/deployment_httpbin.yaml b/db/deployment_httpbin.yaml
+new file mode 100644
+index 0000000..1bd80b5
+--- /dev/null
++++ b/db/deployment_httpbin.yaml
+@@ -0,0 +1,25 @@
++apiVersion: apps/v1
++kind: Deployment
++metadata:
++  labels:
++    app: backend
++    tier: db
++  name: httpbin
++  namespace: staging
++spec:
++  replicas: 4
++  selector:
++    matchLabels:
++      app: backend
++      tier: db
++  template:
++    metadata:
++      labels:
++        app: backend
++        tier: db
++    spec:
++      containers:
++      - image: kennethreitz/httpbin
++        name: httpbin
++        ports:
++        - containerPort: 9876
+diff --git a/db/resources.yaml b/db/resources.yaml
+index c88ef2d..b0d03ce 100644
+--- a/db/resources.yaml
++++ b/db/resources.yaml
+@@ -1,20 +1,19 @@
+-# Copyright 2021 Google LLC
+-#
+-# Licensed under the Apache License, Version 2.0 (the "License");
+-# you may not use this file except in compliance with the License.
+-# You may obtain a copy of the License at
+-#
+-#      http://www.apache.org/licenses/LICENSE-2.0
+-#
+-# Unless required by applicable law or agreed to in writing, software
+-# distributed under the License is distributed on an "AS IS" BASIS,
+-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-# See the License for the specific language governing permissions and
+-# limitations under the License.
+-
```

Comment:
@Shell32-Natsu @mengqiy Can we figure out which function is causing deletion of the license headers ?

Reply:
see comment below